### ISSUE TITLE
Add possibility to apply main indentation rule without any exceptions

### ISSUE THREAD

**Issue by tkachenkomaria244** [sqlfluff/sqlfluff#1284]
For the indentation main rule "For Keywords within a statement, the first root keyword of each line should be aligned" there is an exception now: "if two (or more) of these keywords are on the same line, then the second (and any further) keywords won’t raise a violation, provided that the first was correctly aligned".

Please add the functionality to fix code according to this rule without this exception.
Parameterization seems good to me.

**Comment by barrywhart** [comment]
Can you provide one or more examples of queries and the desired output (after fixes) with your proposed enhancement? That will help ensure we understand your request.

**Comment by barrywhart** [comment]
This issue sounds similar to #1247.

**Comment by tkachenkomaria244** [comment]
> Can you provide one or more examples of queries and the desired output (after fixes) with your proposed enhancement? That will help ensure we understand your request.

For example:
I have script (it is already fixed with L003 rule and others):
```
select
	t1.c1
	, t1.c2
	, tt2.v_sum
from table1 as t1
left join
	(
		select
			t2.c1
			, sum(t2.val1) as v_sum from table2 as t2 group by t2.c1
	) as tt2
	on t1.c1 = tt2.c1
```
but expect to see all keywords in subquery ("from table2 as t2", "group by t2.c1") on the new rows aligned with the first one:
```
select
	t1.c1
	, t1.c2
	, tt2.v_sum
from table1 as t1
left join
	(
		select
			t2.c1
			, sum(t2.val1) as v_sum
		from table2 as t2
		group by t2.c1
	) as tt2
	on t1.c1 = tt2.c1
```

**Comment by alanmcruickshank** [comment]
I think this addressed by the more aggressive reflow code: #3942
